src/openforms/appointments/contrib/qmatic/tests/test_plugin.py

```diff
@@ -353,19 +353,19 @@ def test_get_times_server_error(self, m, status_code):
 
         self.assertEqual(times, [])
 
+    @c_profile()
```

Comment:
this aggregates the profiling for the entire hypothesis run, but it's the only way I can seem to get the data generation included in the profiling